packages/app-backend-core/src/legacy/scan.ts

```diff
@@ -60,6 +60,16 @@ export function scan () {
         // Ignore
       }
     }
+
+    // Scan for Vue instances in the <micro-app> elements
+    const microApps = document.querySelectorAll('micro-app')
```

Comment:
We should probably let users configure this selector in case they use something else than micro-app?

Reply:
Oh, that sounds really flexible~
Maybe I could try to put it into GlobalSettings!

Reply:
Maybe it would be better on `window.VUE_DEVTOOLS_CONFIG` since it's specific to each app. Example:
https://devtools.vuejs.org/guide/open-in-editor.html#customize-request

You can probably use `getPageConfig` from `page-config.ts` in backend core

Reply:
@Akryum I've added it into GlobalSettings. It looks like following images.

![image](https://user-images.githubusercontent.com/16652879/148397213-7580bb7c-8990-42e8-bb23-e4839497ef65.png)

![image](https://user-images.githubusercontent.com/16652879/148397292-20f69122-76ab-4d90-aa12-e6c972063f82.png)

Reply:
> Maybe it would be better on `window.VUE_DEVTOOLS_CONFIG` since it's specific to each app. Example: https://devtools.vuejs.org/guide/open-in-editor.html#customize-request
> 
> You can probably use `getPageConfig` from `page-config.ts` in backend core

Okay I would try it~ Thank you for your advice!

Reply:
I have change to use `getPageConfig ` to get the config and add a section in docs to explain the config, it works great to me!

Reply:
> @Akryum I've added it into GlobalSettings. It looks like following images.
> 
> ![image](https://user-images.githubusercontent.com/16652879/148397213-7580bb7c-8990-42e8-bb23-e4839497ef65.png)
> 
> ![image](https://user-images.githubusercontent.com/16652879/148397292-20f69122-76ab-4d90-aa12-e6c972063f82.png)

The part of this is removed.